charts/linkerd-issuers/Chart.yaml

```diff
@@ -0,0 +1,3 @@
+apiVersion: v2
```

Comment:
Why are we adding a helm chart for the certificates when we already have pure terraform code that creates them?  The terraform code would need enhancements for the additional linkerd helm charts (linkerd-viz, linkerd-jaeger) but I don't see why we abandon the original approach.